net/tstun/wrap.go

```diff
@@ -920,6 +920,9 @@ func (t *Wrapper) Read(buffs [][]byte, sizes []int, offset int) (int, error) {
 		if !t.disableFilter {
 			response := t.filterPacketOutboundToWireGuard(p, pc)
 			if response != filter.Accept {
+				metricOutboundPacketsTotal.Add(trafficLabel{
+					Action: TrafficActionDropACL,
```

Comment:
I think the right place to increment this with `reason="error"` is on line 895 above.